AAI/AAIConnectProfile.md

```diff
@@ -396,6 +397,10 @@ the Broker.
     presented and/or transformed without misrepresenting the original intent,
     except for accommodating for `exp` timestamps to be represented as
     indicated above.
+    
+3.  An Embedded Token Issuer MAY include the `aud` field to indicate which
+    [Brokers](#term-broker) were consented by the user. The values within the `aud`
```

Comment:
"consent" has its own flavor of interpretation that may be both too broad (perhaps in GDPR) and too narrow (in that it doesn't have to involve a user). For the Passport spec, the relationship between the Broker and the Visa Issuer is purposely unspecified except to follow this spec. Agreed, there is a MAY here, but it won't have much meaning if the wording covers a more narrow use case.

Instead of "to indicate which Brokers were consented by the user", I would pivot "to identify the [Brokers](#term-broker) as the intended audience as specified by [RFC 7523 Section 3](https://tools.ietf.org/html/rfc7523#section-3)".

Although this seems more vague, it does provide more coverage for who can make use of this statement such that the network can handle the "how" and "what" the user interaction is that leads to this result.